djstripe/models/payment_methods.py

```diff
@@ -84,18 +86,18 @@ class LegacySourceMixin:
     customer: Optional[StripeForeignKey]
     account: Optional[StripeForeignKey]
 
-    @classmethod
-    def _get_customer_from_kwargs(cls, **kwargs):
-        if "customer" not in kwargs or not isinstance(kwargs["customer"], Customer):
-            raise StripeObjectManipulationException(
-                "{}s must be manipulated through a Customer. "
-                "Pass a Customer object into this call.".format(cls.__name__)
-            )
+    # @classmethod
```

Comment:
You're good to delete stuff in your commits rather than comment them out.

Reply:
Outdated